packages/medusa/src/strategies/batch-jobs/product/export.ts

```diff
@@ -230,7 +230,7 @@ export default class ProductExportStrategy extends AbstractBatchJobStrategy<
         const { writeStream, fileKey, promise } = await this.fileService_
           .withTransaction(transactionManager)
           .getUploadStreamDescriptor({
-            name: `product-export-${Date.now()}`,
+            name: "exports/products/products",
```

Comment:
comment/question: Think we should settle on a naming convention here. I see the order export strategy uses `exports/order/order-export`, which I think is more appropriate than this one. Also, why did we go away from including a timestamp in the name? Sorry if I missed a comment from a previous review, but couldn't seem to find one.

We could do:
`exports/orders/export-${Date.now()}`
`exports/products/export-${Date.now()}`

cc @adrien2p

Reply:
@olivermrbl the timestamp is managed by the plugin itself to avoid conflicting file names. we shouldn't care about naming conflict in the code itself and it is something i ve done for the s3 plugin ages ago if I am not wrong :) so we are following the pattern with other plugins as well wdyt?

But I have one question, @pKorsholm you said that minio support path in the name as it is done here. But how is that treated with plugins that does not support that feature?

Reply:
Yeah, atm the fileservice assigns a timestamp, but we should probably change that to be in the strategies. 

what do you mean how is it treated with plugins that dont support this feature? we dont have a way of know exactly, so  it should be handled according to the capabilities of each fileservice that is implemented

Reply:
The timestamp should be append by the plugin, to allow a user to be able to upload multiple time the same file without overriding it.